README.md

```diff
@@ -10,62 +10,6 @@
 
 ## Introduction
 
-**nf-cmgg/structural** is a bioinformatics best-practice analysis pipeline for calling structural variants from short reads.
+**nf-cmgg/structural** is a bioinformatics best-practice analysis pipeline for calling SVs (structural variants), CNVs (copy number variants) and repeat region expansions from short DNA reads. The pipeline handles the calling of the variants and postprocessing (filtering, annotating...)
```

Comment:
structural variants (SVs), copy number variants (CNVs) and repeat region expansions (RREs)